### PR TITLE
New package: xfce4-whiskermenu-plugin-1.5.0

### PR DESCRIPTION
**Pull request by GiedriusS**
Whisker menu for Xfce4

### CODE REVIEW

**Comment by GiedriusS** [comment]
Mhm, cmake is installed ("=> xfce4-whiskermenu-plugin-1.5.0_1: installing target dependency 'cmake-3.2.2_2' ...") on arm* but cmake isn't found. Weird.

**Comment by xtraeme** [comment]
You got it inverted, swap host and makedepends :)

**Comment by GiedriusS** [comment]
Updated and switched, now builds properly :-)

File: srcpkgs/xfce4-whiskermenu-plugin/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-whiskermenu-plugin'
+pkgname="xfce4-whiskermenu-plugin"
+version="1.5.0"
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=lib"
```
Comment (xtraeme) [inline srcpkgs/xfce4-whiskermenu-plugin/template]: This line is completely redundant.

**Comment by xtraeme** [comment]
Why did you close this?

**Comment by GiedriusS** [comment]
Dunno why it got closed... Will upload the new commit in a minute.

**Pull request merged**